TTS/tts/datasets/dataset.py

```diff
@@ -42,6 +44,19 @@ def string2filename(string):
     return filename
 
 
+def get_audio_size(audiopath):
+    if audiopath[-4:] == ".mp3":
+        audio_info = MP3(audiopath).info
+        return int(audio_info.length * audio_info.sample_rate)
+    elif audiopath[-4:] == ".wav" or audiopath[-5:] == ".flac":
```

Comment:
This won't work correctly for FLACs; they're compressed data.

```
-rw-r--r--@  1 akx  staff  18658921 Mar 26  2023 vocals.flac
-rw-r--r--   1 akx  staff  30728372 Mar 26  2023 vocals.wav
```

IOW, the compression ratio here is ~60% of the original.

Reply:
Indeed, good catch in fact it is dynamic. I have got ~51% for other audio.

Reply:
mutagen library supports flac and wav. I have used it all the audio formats. In that way, we can get the length in a more precise way.